crates/tauri-plugin-holochain-service/android/src/main/java/com/plugin/holochain_service/HolochainService.kt

```diff
@@ -48,76 +49,90 @@ class HolochainService : Service() {
         
         /// Install an app
         override fun installApp(
+            callback: IHolochainServiceCallback,
             request: InstallAppPayloadFfiParcel
-        ): AppInfoFfiParcel {
+        ) {
             Log.d(TAG, "installApp")
-            return runBlocking {
-                AppInfoFfiParcel(runtime!!.installApp(request.fromParcel()))
+            serviceScope.launch(Dispatchers.IO) {
```

Comment:
Instead of blocking the main thread, run async `RuntimeFfi` calls in the `IO` thread. Upon completion, call the callback IPC binder provided.